src/sdk/modules/validators/smartSessionValidator/decorators/index.ts

```diff
@@ -0,0 +1,35 @@
+import type { Chain, Client, Hash, Hex, Transport } from "viem"
+import type { SmartAccount } from "viem/account-abstraction"
+import type { EnableSessionsResponse } from "./Types"
+import { type EnableSessionsParameters, enableSessions } from "./enableSessions"
+import {
+  type UseEnabledSessionParameters,
+  useEnabledSession
+} from "./useEnabledSession"
+export type SmartSessionValidatorActions<
+  TSmartAccount extends SmartAccount | undefined
+> = {
+  enableSessions: (
+    args: EnableSessionsParameters<TSmartAccount> & {
+      signatureOverride?: Hex
+    }
+  ) => Promise<EnableSessionsResponse>
+  useEnabledSession: (
+    args: UseEnabledSessionParameters<TSmartAccount> & {
+      signatureOverride?: Hex
+    }
+  ) => Promise<Hash>
+}
+
+export function smartSessionValidatorActions() {
+  return <TSmartAccount extends SmartAccount | undefined>(
+    client: Client<Transport, Chain | undefined, TSmartAccount>
+  ): SmartSessionValidatorActions<TSmartAccount> => ({
+    enableSessions: (args) => enableSessions(client, args),
```

Comment:
Is the word 'enable' useful here? I think the devex is a bit confusing because of this.

Let's discuss which we think should be the 'default' mode of session creation, and then give the 'other' mode an appropriate term.

I'd lean towards this:
- create with use mode = "createSession"
- use with use mode = "useSession"
- create with enable mode = "createDeferredSession"
- use with enable mode = "useDeferredSession"

Does that terminology work?

Reply:
Or if you pass the mode in as an arg, then can we just call these 
- createSession &
- useSession?

Reply:
I have thought of mode as argument but it's not super clear yet.
for use mode can use above name